pkgs/applications/graphics/inkscape/extensions/silhouette/default.nix

```diff
@@ -0,0 +1,78 @@
+{ fetchFromGitHub
+, lib
+, pkgs
+, python3
+, umockdev
+, writeScript
+}:
+
+let
+  launch-sendto_silhouette = writeScript "sendto_silhouette.sh" ''
+    cd $(dirname $0)
+    ./sendto_silhouette.py "$@"
+  '';
+  launch-silhouette_multi = writeScript "silhouette_multi.sh" ''
+    cd $(dirname $0)
+    ./silhouette_multi.py "$@"
+  '';
+in
+python3.pkgs.buildPythonApplication rec {
+  pname = "inkscape-silhouette";
+  version = "1.28.pre+unstable=2023-06-03"; # Note: there has been lot of as-yet-not-released development since 1.27 was released: https://github.com/fablabnbg/inkscape-silhouette/releases/tag/v1.27
+
+  src = fetchFromGitHub {
+    owner = "fablabnbg";
+    repo = pname;
+    rev = "a1941b6f740466a20dfa116415dd599b1e496302";
+    sha256 = "sha256-CLmSHIhaqtSzPXB1HgHPG+fRW7ktcN3Y3IEWuCVLarw=";
+  };
+
+  patches = [
+    ./interpreter.patch
+  ];
+
+  propagatedBuildInputs = [
+    python3.pkgs.pyusb
+    python3.pkgs.lxml
+    python3.pkgs.inkex
+    python3.pkgs.matplotlib
+  ];
+
+  nativeCheckInputs = [
+    python3.pkgs.pytestCheckHook
+    umockdev
+  ];
+
+  pytestFlagsArray = [
+    "test"
+  ];
+
+  format = "setuptools";
+
+  doCheck = true;
+
```

Comment:
Does it make sense to also include a `pythonImportsCheck` section here?

Reply:
I've never heard of `pythonImportsCheck`, so I don't know. Doing some reading, it sounds like it's a tool for [sanity checking a python package that doesn't have unit tests](https://github.com/jfly/nixpkgs/blob/0ab50cfea23ff108aee2888125f5b9187af8a220/doc/languages-frameworks/python.section.md?plain=1#L726-L727)? But in this case, we *do* have unit tests, so my initial reaction is that it seems like there is not much value in this.